ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CassJavaDriverManagerImpl.java

```diff
@@ -20,15 +25,31 @@
 
     @Override
     public Cluster registerCluster(String clName, String contactPoint, int connections, int port) {
-        return registerCluster(clName,contactPoint,connections,port,null,null);
+        return registerCluster(clName,contactPoint,connections,port,null,null,null,null);
     }
         @Override
-    public Cluster registerCluster(String clName, String contactPoint, int connections, int port, String username, String password) {
+    public Cluster registerCluster(String clName, String contactPoint, int connections, int port, String username, String password, String truststorePath, String truststorePass) {
     
         PoolingOptions poolingOpts = new PoolingOptions()
                                      .setConnectionsPerHost(HostDistance.LOCAL, connections, connections)
                                      .setMaxRequestsPerConnection(HostDistance.LOCAL, 32768);
+        KeyStore ks = null;
```

Comment:
You might want to do null check here for `truststorePath` and `truststorePass` before constructing sslContext to avoid exceptions during non SSL scenarios. 

nit: Good to keep an abstract method for `getSSLContext()` in CassJavaDriverManagerImpl.java and implement in this since the internal implementation can provide their own way of getting SSLContext in their organizations.